src/index.ts

```diff
@@ -551,6 +565,53 @@ export class IdsToolbox {
     return false;
   }
 
+  private async preflightNixStoreInfo(): Promise<void> {
+    this.nixStoreTrust = "unknown";
```

Comment:
We already initialize this to "unknown", do we need to do this here?

Reply:
nice